extensions/typescript-language-features/package.json

```diff
@@ -270,6 +270,100 @@
           "description": "%configuration.suggest.includeAutomaticOptionalChainCompletions%",
           "scope": "resource"
         },
+        "typescript.inlayHints.includeInlayParameterNameHints": {
```

Comment:
Can we remove the duplication of the text `inlay hints` in the setting name?

For example `typescript.inlayHints.parameterNames.enabled`?

This also lets us scope a few of the other settings: `javascript.inlayHints.suppressInlayParameterNameHintsWhenArgumentMatchesName` -> `javascript.inlayHints.parameterNames.suppressWhenArgumentMatchesName`

Reply:
Done.